iogt/templates/base.html

```diff
@@ -41,14 +43,34 @@
       })(window,document,'script','dataLayer','{{settings.core.SiteSettings.ga_tag_manager}}');</script>
       <!-- End Google Tag Manager -->
       {% endif %}
-
+      
+      <div class="languages">
       {% if languages %}
-      <div class="languages l-indent">
-        {% for language in languages %}
-          <a href="{% url 'locale_set' language.locale %}?next={{request.path}}">{{language}}</a>
-        {% endfor %}
-      </div>
+        {% get_current_language as LANGUAGE_CODE %}
+        <div class="language__current">
+          <span class="language__title language__title--current">{% trans "Site Language:" %}  </span>
+          <span class="language__title">{{LANGUAGE_CODE|language_name_local}}</span>
+        </div>
+        <label for="language_dropdown" class="language__dropdown-button">
+          <span class="language__title">Change Language</span>
```

Comment:
1. The `span` tags aren't necessary if we remove the icon, and use `:after`
2. Use `{% trans "Change Language" %}`